fix(analysis): fix and add analysis tables

evansdianga opened the pull request:
This fixes the calculations, wording, rendering and order of various analysis tables

Refs #2

evansdianga committed:
fix(analysis): fix and add analysis tables

This fixes the calculations, wording, rendering and order of various analysis tables

Refs #2

esurface reviewed (approved):
Looks good to me! Thanks for cleaning up the console logs.

Commit:
Update deploy script

evansdianga committed:
fix(analysis): update column names

Change the column names to have consistent naming.

evansdianga committed:
fix((analysis)): add column to follow up table and fix calculations

evansdianga committed:
fix((analysis)): add column to follow up table

evansdianga committed:
fix((analysis)): add column to follow-up table and fix calculations

evansdianga committed:
Add column

evansdianga committed:
feat(entomology-analysis): Create entomology tables

1. Created Routes for Entomology Analysis
2. Updated Menu Items with correct routes
3. Delete `EntomologySpecimensView.coffee` file and create
`EntomologyAnalysisView.coffee` instead
4. Create Model file for specimens
5. Create data fetchers and data transformation logic for getting
entomology data

Commit:
Add buttons for adding geo locations

Commit:
Update deployment script to add protections for pointing to correct db
